guides/common/modules/proc_rex-pull-based-transport-tuning.adoc

```diff
@@ -0,0 +1,33 @@
+[id="REX_pull_based_transport_tuning_{context}"]
+= REX pull-based transport tuning
```

Comment:
```suggestion
= Increasing Host Limit for Pull-Based REX Transport
```
Please, also rename the ID and file name. And watch the capitalization of the title.

Reply:
done. thanks